packages/components/src/item-group/styles.ts

```diff
@@ -6,34 +6,45 @@ import { css } from '@emotion/react';
 /**
  * Internal dependencies
  */
-import { CONFIG, COLORS } from '../utils';
-
-export const unstyledButton = css`
-	appearance: none;
-	border: 1px solid transparent;
-	cursor: pointer;
-	background: none;
-	text-align: start;
-
-	svg,
-	path {
-		fill: currentColor;
-	}
-
-	&:hover {
-		color: ${ COLORS.ui.theme };
-	}
-
-	&:focus-visible {
-		box-shadow: 0 0 0 var( --wp-admin-border-width-focus )
-			var(
-				--wp-components-color-accent,
-				var( --wp-admin-theme-color, ${ COLORS.ui.theme } )
-			);
-		// Windows high contrast mode.
-		outline: 2px solid transparent;
-	}
-`;
+import { CONFIG, COLORS, font } from '../utils';
+
+export const unstyledButton = ( as: 'a' | 'button' ) => {
+	return css`
+		font-size: ${ font( 'default.fontSize' ) };
+		font-family: inherit;
+		appearance: none;
+		border: 1px solid transparent;
+		cursor: pointer;
+		background: none;
+		text-align: start;
+		text-decoration: ${ as === 'a' ? 'none' : undefined };
+
+		svg,
+		path {
+			fill: currentColor;
+		}
+
+		&:hover {
+			color: ${ COLORS.ui.theme };
+		}
+
+		&:focus {
+			box-shadow: none;
+			outline: none;
+		}
+
+		&:focus-visible {
+			box-shadow: 0 0 0 var( --wp-admin-border-width-focus )
+				var(
+					--wp-components-color-accent,
+					var( --wp-admin-theme-color, ${ COLORS.ui.theme } )
+				);
+			// Windows high contrast mode.
+			outline: 2px solid transparent;
+			outline-offset: 0;
```

Comment:
The `outline-offset` was added in this PR and is needed to make the outline size consistent in Windows high contrast mode.